src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/GroupFeedback/GroupFeedbackRow.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  ActionRow, Form, Icon, IconButton, Row,
+} from '@edx/paragon';
+import { DeleteOutline } from '@edx/paragon/icons';
+import PropTypes from 'prop-types';
+import messages from '../../messages';
+
+export const GroupFeedbackRow = ({
+  value,
+  handleAnswersSelectedChange,
+  handleFeedbackChange,
+  handleEmptyFeedback,
+  handleDelete,
+  answers,
+  // injected
+  intl,
+}) => (
+
+  <div className="mb-3">
+    <ActionRow className="mb-1">
+      <Form.Control
+        value={value.feedback}
+        onChange={handleFeedbackChange}
+        onBlur={handleEmptyFeedback}
+      />
+      <ActionRow.Spacer />
+      <IconButton
+        src={DeleteOutline}
+        iconAs={Icon}
+        alt={intl.formatMessage(messages.settingsDeleteIconAltText)}
+        onClick={handleDelete}
+      />
+    </ActionRow>
+    <Form.CheckboxSet
+      onChange={handleAnswersSelectedChange}
+      value={value.answers}
+    >
+      <Row className="mx-0">
+        {answers.map((letter) => (
+          <Form.Checkbox
+            className="mr-3"
```

Comment:
Could you add something like mt-1 or mt-2 for a little bit of top margin?

Reply:
Sure can do.